src/aides.yaml

```
@@ -306,6 +306,52 @@ aides . caen:
   plafond: vélo . prix
   lien: https://caen.fr/velo-pied
 
+aides . mondeville:
+  remplace: commune
+  titre: Ville de Mondeville
+  condition de ressources: oui
+  applicable si:
+    toutes ces conditions:
+      - localisation . code postal = '14120'
+      - une de ces conditions:
+        - vélo . cargo
+        - vélo . électrique
+        - vélo
```

Vu que la condition sur le type de vélo est déjà dans le calcul, on peut la retirer ici

```suggestion
  applicable si: localisation . code postal = '14120'
```